src/components/autocomplete/autocomplete.scss

```diff
@@ -110,9 +110,17 @@ md-autocomplete {
     padding: 0 15px;
     line-height: 40px;
     height: 40px;
+    position: absolute;
+  }
+  .md-autocomplete-placeholder {
+    color: #757575;
+    font-size: 14px;
+    line-height: 40px; // Matches md-autocomplete height.
+    padding: 0 15px;
   }
   .md-show-clear-button button {
-    position: relative;
+    position: absolute;
+    right: 0;
```

Comment:
What prompted this change (and below)? The pull request description doesn't make that clear.